### DISCUSSION TITLE
Calculating orientations for faults returning error

### DISCUSSION THREAD

**Discussion by andimg122** [gempy-project/gempy#587] [Q&A]
**What is your question?**
Hello everyone,
I have 4 sets of fault data that I wish to calculate the orientations for, however when doing so I keep receiving the error message **"ValueError: The values of normal are not valid."** for two of them. the other two seem to calculate just fine. 

Any help is greatly appreciated, thank you.

**Comment by AlexanderJuestel** [comment]
Hey @andimg122, 

thanks for your question. Hopefully we can help you out again! Could you provide us with a sample of the fault data that is working and that fails? Which calculations are you performing to obtain the orientation values? Which functions do you use?

If you provide us with some more information, it is easier for us to find the issue and to solve it ;) 

Cheers Alex

**Comment by andimg122** [comment]
Hi @AlexanderJuestel ,

I use the function "set_orientation_from_surface_points" to calculate the orientations. I've attached two datasets, one working and one not working.

[Faults.zip](https://github.com/cgre-aachen/gempy/files/4688505/Faults.zip)
Thank you for your kind assistance!

**Comment by andimg122** [comment]
Hi @AlexanderJuestel , were you able to recreate the issue with the dataset? It's very important that I solve this.

Best regards

**Comment by AlexanderJuestel** [comment]
Hey @andimg122 ,

unfortunately, I have not had the time to look at your issue as I am quite busy these days with my thesis. Sorry about that. Maybe someone else from team may be able to help.

Cheers 
Alex
